src/components/NewCommentForm/helper.ts

```diff
@@ -0,0 +1,22 @@
+import { Form } from '../../types/Form';
+
+export const validateForm = (
```

Comment:
You don't need to move it outside the component. 
The main idea is to split your logic, so it will be easier to maintain in the future. So example you need to change validation, so you just go to validate function and do it instead of looking for validation in the form handler